docs/src/main/asciidoc/logging.adoc

```diff
@@ -672,6 +666,82 @@ implementation("org.jboss.slf4j:slf4j-jboss-logmanager")
 
 . Verify whether the logs generated by the added library adhere to the same format as the other Quarkus logs.
 
+=== Using MDC (Mapped Diagnostic Context)
```

Comment:
Also not an imperative verb phrase, does that matter?

Reply:
Intiailly I used `Use MDC`. But it reads strange, and @geoand proposed `Using`. 
I'm not totally sure what's the best.

Reply:
Where is the guidance for this coming from? Frankly it seems like a really weird rule.

Reply:
Never mind, I see the link above. It's easy to miss things on the github phone app :(

Reply:
"Use MDC" is a bit terse, but can we add a purpose? For example "Use MDC to add contextual log information", and then define MDC in the paragraph below?

Reply:
Changed it to "Use MDC to add contextual log information"